src/lib/spinel/spinel.h

```diff
@@ -3909,6 +3914,34 @@ enum
 
     SPINEL_PROP_CNTR__END = 0x800,
 
+    SPINEL_PROP_RCP__BEGIN = 0x800,
+
+    /// MAC Key
+    /** Format: `Cddd`.
+     *
+     *  `C`: MAC key ID mode
+     *  `d`: previous MAC key material data
+     *  `d`: current MAC key material data
+     *  `d`: next MAC key material data
+     *
+     * The Spinel property is used to set/get MAC key materials to and from RCP.
```

Comment:
Just curious - 
1) there is only set handler in the implementation, will this be a read/write property or a write-only property?

2) I see there are some benefits of splitting KeyId from the Keys, but if `SPINEL_PROP_RCP_MAC_KEY` and `SPINEL_PROP_RCP_MAC_KEY_ID` are write-only, maybe combining key and key id may reduce the amount of APIs?

Reply:
1. Currently it is a write only property. Host does not have the need to get MAC key from RCP for now. We can implement it later if needed. I would prefer to keep it as write only as MAC keys are sensitive information.

2. This is out of naming and future expansion consideration. What if we need to read this property from RCP later or need to change only Key ID.

Reply:
MAC keys are derived from masterkey and sequence number, so if sequence number (key id) changes, MAC keys should always change as well, right?

Reply:
That is true. I was also considering expanding this property to be used for Key ID mode 0 and 2. For mode 0 case, the Key ID is not there.

Reply:
On a second thought, combining MAC key and MAC key ID into one property might help address the racing condition when MAC key is updated and Key ID is not, an Enh-ACK is transmitted in between with wrong key used. I will change the implementation.